test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java

```diff
@@ -139,6 +139,28 @@ public static String getMsiProperty(JPackageCommand cmd, String propertyName) {
         .executeAndGetOutput().stream().collect(Collectors.joining("\n"));
     }
 
+    public static String getExecutableDesciption(Path pathToExeFile) {
+        String description = null;
+        Executor exec = Executor.of("powershell",
+                "-NoLogo",
+                "-NoProfile",
+                "-Command",
+                "(Get-Item \\\""
+                + pathToExeFile.toAbsolutePath()
+                + "\\\").VersionInfo | select FileDescription");
+        List<String> lines = exec.executeAndGetOutput();
+        for (int i = 0; i < lines.size(); i++) {
+            if (lines.get(i).trim().equals("FileDescription")) {
+                i += 2; // Skip "---------------" and move to description
+                description = lines.get(i).trim();
+            }
+        }
+
+        TKit.assertNotNull(description, "Failed to get file description");
```

Comment:
Failure to get the executable's description through powersehll script is not an issue of a test case being executed. This is the testing framework issue. Throwing RuntimeException with the message containing file path will be the appropriate error handling. Having a file name in the exception message will help to localize the issue.

Reply:
Fixed. Error message also fixed for description check.